packages/react-tabster/src/hooks/useArrowNavigationGroup.ts

```diff
@@ -17,6 +18,8 @@ export interface UseArrowNavigationGroupOptions {
  * @param options - Options to configure keyboard navigation
  */
 export const useArrowNavigationGroup = (options?: UseArrowNavigationGroupOptions) => {
+  useTabster();
```

Comment:
This looks extremely suspicious 🤔 Is it really required?

Reply:
That's the problem with using a hook implem, you need to guarantee tabster exists on the page

Reply:
even if you only use DOM attributes, tabster needs to be created at some point

Reply:
I think that it's something where we should make a follow up 🙄

- Tabster requires `window` to be created
- `getTabsterAttribute()` requires Tabster instance
- SSR does not have `window` 💣

_(do I correctly get?)_

IMO, `getTabsterAttribute()` should not depend on Tabster instance.